frontend/packages/dev-console/src/plugin.tsx

```diff
@@ -164,6 +166,22 @@ const plugin: Plugin<ConsumedExtensions> = [
       required: [FLAGS.OPENSHIFT],
     },
   },
+  {
+    type: 'NavItem/Href',
+    properties: {
+      perspective: 'dev',
+      group: 'top',
+      componentProps: {
+        name: 'GitOps',
+        namespaced: true,
```

Comment:
Gitops page is actually not a namespaced page.
```suggestion
        namespaced: false,
```